_docs/setup/kubernetes/ansible-install.md

```diff
@@ -1,122 +1,109 @@
 ---
 title: Installation with Ansible
-overview: Install Itio with the included Ansible playbook.
+overview: Install Istio with the included Ansible playbook.
 
 order: 40
 
 layout: docs
 type: markdown
 ---
-{% include home.html %}
 
-The Ansible scenario defined within this project will allow you to : 
+{% include home.html %}
 
-- Deploy Istio on Kubernetes or Openshift by specifying different parameters (version, enable auth, deploy bookinfo, ...)
-- Specify the addons to be deployed such as `Grafana`, `Prometheus`, `Servicegraph`, `Zipkin` or `Jaeger`
+Instructions for the installation and configuration of Istio using Ansible.
 
 ## Prerequisites
 
-- [Ansible 2.4](https://docs.ansible.com/ansible/latest/intro_installation.html)
+The following instructions require [Ansible 2.4](https://docs.ansible.com/ansible/latest/intro_installation.html). Additionally Kubernetes **1.7.3 or newer** is required.
+
+The following prerequisites must be met if using Openshift.
```

Comment:
OpenShift (uppercase S), here and elsewhere.

Reply:
fixed.